components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.ts

```diff
@@ -1,139 +1,80 @@
-import { Component, EventEmitter, Input, Output, OnChanges, OnInit, SimpleChanges } from '@angular/core';
```

Comment:
Better **not** to look at the diff on this file--all the old text now comprises the parent class. The contents of this file is all new, having just the little bit needed for the subclass.